kms/apiv1/key_management_client.go

```diff
@@ -63,9 +62,6 @@ type KeyManagementCallOptions struct {
 	UpdateCryptoKeyPrimaryVersion []gax.CallOption
 	DestroyCryptoKeyVersion       []gax.CallOption
 	RestoreCryptoKeyVersion       []gax.CallOption
-	GetIamPolicy                  []gax.CallOption
```

Comment:
@noahdietz I am not seeing what would have triggered these removals in cl 379309757.... Any ideas?

Reply:
I will look into it.

Reply:
Foudn the problem - it's a change in the Service config YAML...fixing.